boards/arm/olimexino_stm32/Kconfig.defconfig

```diff
@@ -44,4 +44,20 @@ config PWM_STM32_1
 
 endif # PWM
 
+if USB
+
+config USB_DC_STM32_DISCONN_ENABLE
+	default y
+
+config USB_DC_STM32_DISCONN_GPIO_PORT_NAME
```

Comment:
Can you open an issue to convert this to use DT GPIO

Reply:
I will not open an issue because I had already this in mind. I just waited for this PR to get merged first.